src/editorial/web/components/lines/squiggly.ts

```diff
@@ -1,2 +1,18 @@
+import { border } from "@guardian/src-foundations/palette"
+
+const squigglySvg = encodeURIComponent(`
+<svg
+	xmlns="http://www.w3.org/2000/svg" width="12" height="15"
+	viewBox="0 0 12 15"
+	xmlns:xlink="http://www.w3.org/1999/xlink"
+>
+	<g stroke-width="1" stroke="${border.secondary}" fill="none">
+		<path id="squiggle" d=" M 0 0.5 q 1.5 0 3 1.5 t 3 1.5 t 3 -1.5 t 3 -1.5" />
```

Comment:
As we discussed, would you be able to paramterise this curve to make it:

1. easier to understand
2. easier to update in the future

```tsx
<path id="squiggle" d=" M 0 0.5 q 1.5 0 ${frequency} ${amplitude/2} t ${frequency} ${amplitude/2} t ${frequency} -${amplitude/2} t ${frequency} -${amplitude/2}" />
```

Reply:
Great, just updated!